docs/4.0/components/forms.md

```diff
@@ -337,7 +337,10 @@ More complex layouts can also be created with the grid system.
     </div>
     <div class="form-group col-md-4">
       <label for="inputState" class="col-form-label">State</label>
-      <select id="inputState" class="form-control">Choose</select>
+      <select id="inputState" class="form-control">
+        <option selected>Choose...</option>
```

Comment:
The first option the attribute `selected` is not necessary.

Reply:
sure, but also not harmful (and matches the existing use of `selected` in another example later on in this document)